<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginDialog</class>
 <widget class="QWidget" name="LoginDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>707</width>
    <height>479</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>651</width>
     <height>441</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_1">
    <attribute name="title">
     <string>登录</string>
    </attribute>
    <widget class="QLabel" name="lb_name">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>用户名</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_password">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>160</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>密码</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_login">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>280</y>
       <width>111</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>登录</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_clear">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>280</y>
       <width>131</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>清空</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>30</y>
       <width>391</width>
       <height>191</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLineEdit" name="le_tel">
        <property name="text">
         <string>15230296211</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="le_password">
        <property name="text">
         <string>291314585air+air</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>注册</string>
    </attribute>
    <widget class="QPushButton" name="pb_regsiter">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>340</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>注册</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_clear_2">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>340</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>清空</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>210</y>
       <width>541</width>
       <height>111</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QCheckBox" name="cb_dance">
      <property name="geometry">
       <rect>
        <x>421</x>
        <y>41</y>
        <width>57</width>
        <height>21</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>57</width>
        <height>21</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>57</width>
        <height>21</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>舞蹈</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="cb_food">
      <property name="geometry">
       <rect>
        <x>73</x>
        <y>41</y>
        <width>57</width>
        <height>21</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>57</width>
        <height>21</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>57</width>
        <height>21</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>美食</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="cb_funny">
      <property name="geometry">
       <rect>
        <x>189</x>
        <y>41</y>
        <width>57</width>
        <height>21</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>57</width>
        <height>21</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>57</width>
        <height>21</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>搞笑</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="cb_ennegy">
      <property name="geometry">
       <rect>
        <x>305</x>
        <y>41</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>57</width>
        <height>21</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>111111</width>
        <height>111111</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>正能量</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="cb_music">
      <property name="geometry">
       <rect>
        <x>73</x>
        <y>71</y>
        <width>57</width>
        <height>21</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>57</width>
        <height>21</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>57</width>
        <height>21</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>歌曲</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="cb_video">
      <property name="geometry">
       <rect>
        <x>189</x>
        <y>71</y>
        <width>57</width>
        <height>21</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>57</width>
        <height>21</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>57</width>
        <height>21</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>影视</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="cb_outdoor">
      <property name="geometry">
       <rect>
        <x>305</x>
        <y>71</y>
        <width>57</width>
        <height>21</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>57</width>
        <height>21</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>57</width>
        <height>21</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>户外</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="cb_educate">
      <property name="geometry">
       <rect>
        <x>421</x>
        <y>71</y>
        <width>57</width>
        <height>21</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>57</width>
        <height>21</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>57</width>
        <height>21</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>教育</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="cb_all">
      <property name="geometry">
       <rect>
        <x>71</x>
        <y>11</y>
        <width>57</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>全选</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>30</y>
       <width>441</width>
       <height>175</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLineEdit" name="le_resiger_name">
        <property name="text">
         <string>刘洋</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="le_resiger_tel">
        <property name="text">
         <string>15230296211</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="le_resiger_password">
        <property name="text">
         <string>291314585air+air</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="le_resiger_confirm">
        <property name="text">
         <string>291314585air+air</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>111</width>
       <height>171</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>用户名</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lb_resiger_tel">
        <property name="text">
         <string>手机号</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lb_resiger_password">
        <property name="text">
         <string>密码</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lb_resiger_confirm">
        <property name="text">
         <string>确认密码</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
